scripts/cangen/main.py

```diff
@@ -68,6 +68,14 @@ def parse():
     bus_list = config["canGen"]["busses"]
     output_path = config["canGen"].get("outputPath", DEFAULT_OUTPUT_DIR)
 
+
+#clear the directory file
+if os.path.exists(output_path):
```

Comment:
this whole block should be indented to be inside the `'if __name__ == "__main__":` block